prow/config.yaml

```diff
@@ -5761,6 +5761,47 @@ presubmits:
           requests:
             memory: "2Gi"
 
+  - name: pull-test-infra-verify-config-canary
+    agent: kubernetes
+    context: pull-test-infra-verify-config-canary
+    branches:
+    - master
+    rerun_command: "/test pull-test-infra-verify-config-canary"
+    trigger: "/test pull-test-infra-verify-config-canary"
+    path_alias: "k8s.io/test-infra"
+    decorate: true
+    utility_images:
+      clonerefs: "registry.svc.ci.openshift.org/ci/clonerefs:latest"
```

Comment:
@BenTheElder can we get builds for these images set up on `gcr.io`?

Reply:
do you want to add some image push rules for them?

Reply:
Looks like the rules exist, just have not been used. We can continue to serve from our registry until these are more stable.